jwst/extract_1d/extract_1d_step.py

```diff
@@ -149,6 +159,8 @@ class Extract1dStep(Step):
     apply_apcorr = boolean(default=True)  # apply aperture corrections?
     ifu_autocen = boolean(default=False) # Auto source centering for IFU point source data.
     ifu_rfcorr = boolean(default=False) # Apply 1d residual fringe correction
+    ifu_set_srctype = option("POINT", "EXTENDED", None, default=None) # user-supplied source type
+    ifu_rscale = float(default=None, min=0.5, max=3) Radius in terms of PSF FWHM to scale extraction radii
```

Comment:
```suggestion
    ifu_rscale = float(default=None, min=0.5, max=3) # Radius in terms of PSF FWHM to scale extraction radii
```

Reply:
I think it would be useful to add a little if/then block down around line 230 in this top-level step module (after all the checking is done on input datamodel types) so that a log warning message is issued if either `ifu_set_srctype` or `ifu_rscale` have been set by the user and `meta.exposure.type != "MIR_MRS"` to let them know that the values they supplied don't apply to this exp_type and will be ignored (rather than just ignoring them silently).

Reply:
good suggestion. I will do that